scenes/Restaurant/Customer.gd

```diff
@@ -23,6 +23,10 @@ onready var _munch_sounds:Array = [
 	$Munch4
 ]
 
+# the food object which should be animated and recolored when we eat
+onready var Food = get_node("../KludgeCustomer/KludgeNeck/KludgeHead/Food")
+onready var FoodLaser = get_node("../KludgeCustomer/KludgeNeck/KludgeHead/FoodLaser")
```

Comment:
This change is a little surprising, make sure to mention it in the commit comment.

Reply:
Commit comment now mentions "bug where food was being drawn behind the table".